swan-cern/files/swan_hpc_config.py

```diff
@@ -0,0 +1,73 @@
+from kubernetes_asyncio.client.models import (
```

Comment:
After the last restructuring to unite condor with spark, shouldn't this go in [here](https://github.com/swan-cern/swan-charts/blob/master/swan-cern/files/swan_computing_config.py)?

Reply:
Done, can you review and eventually merge? I need your help to config the gitops repo.
(This also needs some testing, as I didn't test the move to swan_computing yet...)